tools/code_review_tools/taskcluster.py

```diff
@@ -185,4 +185,4 @@ def create_blob_artifact(
     )
 
     # Build the absolute url
-    return f"https://queue.taskcluster.net/v1/task/{task_id}/runs/{run_id}/artifacts/{path}"
+    return f"https://community-tc.services.mozilla.com/api/queue/v1/task/{task_id}/runs/{run_id}/artifacts/{path}"
```

Comment:
This should be firefox-ci

Reply:
Yes, this is used to upload patches from code-review bot tasks, so in firefox ci.